blessed/sequences.py

```diff
@@ -189,7 +189,7 @@ def _wrap_chunks(self, chunks):
                     break
                 cur_line.append(chunks.pop())
                 cur_len += chunk_len
-            if chunks and Sequence(chunks[-1], term).length() > width:
+            if chunks and chunk_len > width:
```

Comment:
at this time ``chunk_len`` represents the length of the last chunk that was "popped" and not the last chunk in chunks, but who knows, maybe that's a good thing? just need to add more tests with #273 and if it works then that's fine

Reply:
I was expecting the linter to complain since ``chunk_len`` is defined within the while loop. I'm not sure if it's smart enough to know it's guarded or what, but it's not complaining.

Reply:
I don't think the linter will catch it. mypy will complain about it - but that's because mypy doesn't understand the `if chunks` aspect

Reply:
mypy ran fine

Reply:
As for the dubious use of `chunk_len` outside of while loop, I've opted to incorporate the performance enhancement but move it to the **inside** of while loop in PR #275, thanks!